controller/api.py

```diff
@@ -5,34 +5,51 @@
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
 
 import pn532
-import prx_3hc
+import wiegand
 import store
 import payment_processor
 
 
 class Controller(object):
     def __init__(self):
-        self.prx_3hc = prx_3hc.PRX_3HC()
+        self.wiegand = wiegand.Wiegand26()
         self.pn532 = pn532.PN532()
         self.store = store.Store()
         self.payment_processor = payment_processor.Payment_Processor()
 
     def setup(self):
         print("controller is setting up")
 
-        self.prx_3hc.setup()
+        self.wiegand.setup()
         self.pn532.setup()
         self.store.setup()
         self.payment_processor.setup()
 
+    
+    def get_card(self):
```

Comment:
if this is not gonna be used, it doesn't make sense to include it here. it's up to the controller to decide the error handling in this case.

Reply:
I put this here to show how the error handling worked. didn't expect to keep it. Then again, I should have implemented an example so heavily

Reply:
if you want to show an example, what you could do instead is to create an example file and put it there. you don't want to mix it with your core api.